### PR TITLE
doc: Show usage of the new simpler APIs for mTLS setup coming in Akka HTTP

### PR DESCRIPTION
**Pull request by johanandren**
Quite a bit cleaner, manually tested (I don't think these are part of any actual test)

### CODE REVIEW

File: docs/src/main/paradox/mtls.md [hunk 1]
```diff
@@ -27,11 +29,13 @@ Java
 When run the server will only accept client connections that use a keypair that it considers valid, other connections will be denied
 and fail with a TLS protocol error.
 
+It is possible to rotate the certificates without restarting the service, see @extref[Akka HTTP documentation](akka-http:server-side/server-https-support.html#rotating-certificates) 
+
 
 ## Setting the client up
 
 In the client, the trust store must be set up to trust the server cert, in our sample it is signed with the same CA as the
```
Comment (ennru) [inline docs/src/main/paradox/mtls.md:37]:
```suggestion
In the client, the trust store must be set up to trust the server certificate, in our example it is signed with the same CA as the
```

File: docs/src/main/paradox/mtls.md [hunk 2]
```diff
@@ -40,4 +44,17 @@ Java
...
+It is possible to rotate the certificates without restarting the service, see @extref[Akka HTTP documentation](akka-http:client-side/client-https-support.html#convenient-loading-of-key-and-certificate)
+
+## Further limiting of access using client certificate identities
+
+In addition to requiring a trusted certificate it is possible to further limit access based on the identity present in
+the trusted client certificate ip or dns SAN (Subject Alternative Names) or CN (Common Name).
+
+This is done by wrapping the service handler in the `requireClientCertificateIdentity`:
+
+FIXME provide sample snippets
```
[the remark this answers is not in the document]
Comment (johanandren) [reply docs/src/main/paradox/mtls.md:58]: Added sample in [5e43b17](https://github.com/akka/akka-grpc/pull/1910/commits/5e43b17841908ddf524d4ee1de6b181d91424b5b)

[the remark this answers is not in the document]
Comment (johanandren) [reply docs/src/main/paradox/mtls.md:65]: Wrong commit, sample added in [1b0a1a5](https://github.com/akka/akka-grpc/pull/1910/commits/1b0a1a56e6a26861075d46c5f5561eea59e05cb2)

**Review by patriknw — APPROVED** [review]
LGTM

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**